docs/integrations/mango.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Mango Markets
+head:
+  - - meta
+    - name: title
+      content: Solana Cookbook | Building on Mango Markets
+  - - meta
+    - name: og:title
+      content: Solana Cookbook | Building on Mango Markets
+  - - meta
+    - name: description
+      content: Mango Markets offers the industry standard for decentralized, cross-margin trading. Learn how to use and build on top of Mango Markets.
+  - - meta
+    - name: og:description
+      content: Mango Markets offers the industry standard for decentralized, cross-margin trading. Learn how to use and build on top of Mango Markets.
+  - - meta
+    - name: og:image
+      content: https://solanacookbook.com/cookbook-sharing-card.png
+  - - meta
+    - name: og:image:alt
+      content: Solana splash card
+  - - meta
+    - name: twitter:card
+      content: summary
+  - - meta
+    - name: twitter:site
+      content: "@solanacookbook"
+  - - meta
+    - name: twitter:image
+      content: "https://solanacookbook.com/cookbook-sharing-card.png"
+  - - meta
+    - name: robots
+      content: index,follow,noodp
+  - - meta
+    - name: googlebot
+      content: index,follow
+---
+
+# Mango Markets
+
+Mango provides a single venue to lend, borrow, swap, and leverage 
+trade cryptoassets through an on-chain risk engine.
+You can connect to Mango's on-chain program using the Client API libraries.
+You'll also need the Solana javascript API library.
+
+<SolanaCodeGroup>
+  <SolanaCodeGroupItem title="TS" active>
```

Comment:
Since there is not really a preview here, pressing the switcher button breaks your example. 

Try using `CodeGroup` instead like it is done in [token](https://github.com/solana-developers/solana-cookbook/blob/master/docs/references/token.md)